src/components/taglib/TransformHelper/assignComponentId.js

```diff
@@ -19,24 +19,7 @@ module.exports = function assignComponentId(isRepeated) {
     let assignedKey;
     var nestedIdExpression;
     var idExpression;
-
-    if (!this.hasBoundComponentForTemplate()) {
-        // We are assigning a component ID to a nested component in a template that does not have a component.
-        // That means we do not have access to the parent component variable as part of a closure. We
-        // need to look it up out of the `out.data` map
-        if (!context.isFlagSet('hasComponentVar')) {
-            context.setFlag('hasComponentVar');
-
-            var getCurrentComponentVar = context.importModule('marko_getCurrentComponent',
-                this.getMarkoComponentsRequirePath('marko/components/taglib/helpers/getCurrentComponent'));
```

Comment:
Can `marko/components/taglib/helpers/getCurrentComponent.js` be deleted now? I don't think it's being used anymore.

Reply:
Yes, we can delete it. Good catch.